cfg/sym1.cfg

```diff
@@ -0,0 +1,46 @@
+# sym1-4k.cfg   (4k)
+#
+# for Sym-1 with 4kb RAM
+#
+# ld65 --config sym1-4k.cfg -o <prog>.bin <prog>.o
+
+FEATURES {
+    STARTADDRESS: default     = $0200;
+    CONDES:    segment        = STARTUP,
+               type           = constructor,
+               label          = __CONSTRUCTOR_TABLE__,
+               count          = __CONSTRUCTOR_COUNT__;
+    CONDES:    segment        = STARTUP,
+               type           = destructor,
+               label          = __DESTRUCTOR_TABLE__,
+               count          = __DESTRUCTOR_COUNT__;
+}
+
+SYMBOLS {
+    __STACKSIZE__:      type  = weak,   value = $0080;   # 128 byte program stack
+    __STARTADDRESS__:   type  = export, value = %S;
+}
+
+MEMORY {
+	ZP:		start = $0000,  size = $00F7,                      define = yes,   file = %O;
```

Comment:
Please don't ise TAB chars.

I'm seeing TAB chars on these files of you:

cfg/sym1.cfg
cfg/sym1-4k.cfg
cfg/sym1-32k.cfg
samples/symIO.c
samples/tutorial/sym1/symIO.c
samples/tutorial/sym1/symNotepad.c
samples/tutorial/sym1/symDisplay.c
samples/symNotepad.c
samples/symDisplay.c

Reply:
I agree completely.  I hate TAB characters in source files 'cause it skews the display compared to lines with space chars only, making viewing ugly and difficult.

I'll modify those files to remove TABs and push them to my branch.

Reply:
--- TABs removed ---